controllers/multiclusterhub_controller.go

```diff
@@ -551,6 +554,102 @@ func (r *MultiClusterHubReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
+func (r *MultiClusterHubReconciler) applyTemplate(ctx context.Context, m *operatorv1.MultiClusterHub, template *unstructured.Unstructured) (ctrl.Result, error) {
+	log := log.FromContext(ctx)
+	// Set owner reference.
+	if (template.GetKind() == "ClusterRole") || (template.GetKind() == "ClusterRoleBinding") || (template.GetKind() == "ServiceMonitor") || (template.GetKind() == "CustomResourceDefinition") {
+		utils.AddInstallerLabel(template, m.Name, m.Namespace)
+
+	}
+
+	if template.GetKind() == "APIService" {
+		result, err := r.ensureUnstructuredResource(m, template)
+		if err != nil {
+			log.Info(err.Error())
+			return result, err
+		}
+	} else {
+		// Apply the object data.
+		force := true
+		err := r.Client.Patch(ctx, template, client.Apply, &client.PatchOptions{Force: &force, FieldManager: "multiclusterhub-operator"})
+		if err != nil {
+			log.Info(err.Error())
+			return ctrl.Result{}, pkgerrors.Wrapf(err, "error applying object Name: %s Kind: %s", template.GetName(), template.GetKind())
+		}
+	}
+	return ctrl.Result{}, nil
+}
+
+func (r *MultiClusterHubReconciler) ensureInsights(ctx context.Context, m *operatorv1.MultiClusterHub, images map[string]string) (ctrl.Result, error) {
+
+	log := log.FromContext(ctx)
+
+	templates, errs := renderer.RenderChart("/charts/toggle/insights", m, images)
```

Comment:
I think we decided we want to use an environment variable rather than hard path to templates

Reply:
The code uses an environment variable `TEMPLATES_PATH ` as a prefix in order to locate the charts, but rendering things is still going to require a specific path to the designated chart. I have added that as a constant in the  utils.go